examples/web/index.html

```diff
@@ -1,8 +1,6 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <base href="/flame/">
```

Comment:
```suggestion
  <base href="/">
```

Here we should leave only / instead of removing the tag altogether